### DISCUSSION TITLE
Fix onboarding config file name logging

### DISCUSSION THREAD

**Discussion by aentwist** [renovatebot/renovate#26017] [Suggest an Idea]
## Expected

onboarding config file name is logged

https://github.com/renovatebot/renovate/blob/f4f7a899aed37d92440763dc8ced24808b997eb3/lib/workers/repository/init/apis.ts#L33

## Actual

renovate.json is always logged

https://github.com/renovatebot/renovate/blob/f4f7a899aed37d92440763dc8ced24808b997eb3/lib/workers/repository/init/apis.ts#L45

## Proposed Solution

```typescript
const renovateConfigFileName = defaultConfigFile(config);
const renovateConfig = await getJsonFile(renovateConfigFileName); 
```

```typescript
"Global config disables Renovate - checking ${renovateConfigFileName} to see if it is re-enabled",
```

**Comment by rarkins** [comment]
PR welcome
